docs/datamodel/databases.rst

```diff
@@ -3,3 +3,14 @@
 =========
 Databases
 =========
+
+An EdgeDB deployment can have multiple databases in it.
+:eql:stmt:`CREATE DATABASE` EdgeQL command adds a new database to the
+EdgeDB cluster. This also creates a special ``schema::Database``
```

Comment:
This is oddly specific.  In fact, no special objects are created, since `schema::Database` is a view into a system catalog (which is also not a real table).  Simply say that it is possible to get a list of all databases with an example.

Reply:
I've removed all the "object" talk and left just the sample query.